dependabot.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      # Updates are scheduled at the beginning of the work week.
+      interval: "weekly"
+      day: "Monday"
+      time: "02:00"
+      # US Pacific Time
+      timezone: "America/Los_Angeles"
+    commit-message:
+      prefix: "[dependabot]"
+      reviewers:
+        - "timothytrippel"
```

Comment:
We can probably work with @mundaym in defining a rotation to review these type of changes. I am not sure if dependabot supports GitHub teams? In the meantime, I'd also add someone from lowRISC here and below.

Reply:
@timothytrippel  oops, I missed your previous comment. Feel free to resolve this one.